portability-spi-cloud/src/main/java/org/dataportabilityproject/spi/cloud/types/JobAuthorization.java

```diff
@@ -83,11 +87,14 @@ private static Builder create() {
     @JsonProperty("encryptedImportAuthData")
     public abstract Builder setEncryptedImportAuthData(String authData);
 
+    @JsonProperty("encryptedSymmetricKeyForAuthData")
```

Comment:
The naming of the 3 keys is getting confusing to me.. I think we should either consistently name them according to type (symmetric vs asymmetric) or purpose (session vs worker), or both

what about one of these conventions:
- sessionPrimarySymmetricKey, sessionSecondarySymmetricKey, workerPublicKey
- sessionSymmetricKeyA, sessionSymmetricKeyB, workerPublicKey

and I like the 'forAuthData' but it makes the name pretty long, maybe it could go in the names like this:
- sessionSymmetricAuthDataKeyA, sessionSymmetricAuthDataKeyB, workerPublicAuthDataKey

or don't include it in the field names but have it in a comment instead, "these three keys are all used for encrypting auth data"

Reply:
Ok, renamed to a simpler approach. I think it's wroth documenting and adding a link.